client/pages/project-edit-modal/modal.template.html.slim

```diff
@@ -17,6 +17,11 @@ form novalidate=true role='form' name='projectForm' ng-submit='projectForm.$vali
       p.help-block
         | The is the name of the project as shown in ProbeDock.
     .form-group
+      label for='projectRepoUrl' Repository URL
+      input#projectRepoUrl.form-control type='string' placeholder='{{ editedProject.repoUrl || "Project repository" }}' maxlength=100 ng-model='editedProject.repoUrl'
```

Comment:
Why is `editedProject.repoUrl` in the placeholder? As far as I can see it doesn't do anything, as when the placeholder is shown, it means that `editedProject.repoUrl` is empty anyway.